### PR TITLE
Integrating Ad.Style to AMP

### PR DESCRIPTION
**Pull request by leonidvolinski**
This is the integration of Ad.Style to AMP.
Contributor License Agreement already signed.

### CODE REVIEW

**Comment by leonidvolinski** [comment]
Friendly ping for a review. Thank you!

File: ads/_config.js [hunk 1]
```diff
@@ -193,6 +193,8 @@ const adConfig = jsonConfiguration({
     preconnect: 'https://ad.ad-stir.com',
   },
 
+  'adstyle': {},
```
Comment (calebcordry) [inline ads/_config.js]: I see your not using renderStart here, we recommend implementing it if possible, should help with UX and get some performance gains. It is optional.

**Comment by calebcordry** [comment]
@leonidvolinski Could you also add your network to `extensions/amp-ad/amp-ad.md`? Thanks!

File: extensions/amp-ad/amp-ad.md [hunk 2]
```diff
@@ -271,6 +271,7 @@ See [amp-ad rules](https://github.com/ampproject/amphtml/blob/master/extensions/
 - [Aniview](../../ads/aniview.md)
 - [AppNexus](../../ads/appnexus.md)
 - [AppVador](../../ads/appvador.md)
+- [AdStyle](../../ads/adstyle.md)
```
Comment (calebcordry) [inline extensions/amp-ad/amp-ad.md]: nit: please sort alphabetically.

File: ads/_config.js [hunk 3]
```diff
@@ -193,6 +193,12 @@ const adConfig = jsonConfiguration({
     preconnect: 'https://ad.ad-stir.com',
   },
 
+  'adstyle': {
+    renderStartImplemented: true,
```
Comment (calebcordry) [inline ads/_config.js]: Great that you decided to implement this, but I do not see this callback being executed. Can you confirm you are calling this in you external JS? If you are unable to do so you should remove this flag (keep the prefetch and preconnect) as you will suffer a small penalty if the runtime is waiting for this signal but it is never called.

**Pull request merged**